gfx.cpp

```diff
@@ -118,7 +118,9 @@ class GfxInternal
     ID3D12CommandSignature *draw_mesh_signature_ = nullptr;
     std::vector<D3D12_RESOURCE_BARRIER> resource_barriers_;
     ID3D12Resource **back_buffers_ = nullptr;
+    D3D12MA::Allocation **back_buffer_allocations_ = nullptr;
     uint32_t *back_buffer_rtvs_ = nullptr;
+    bool is_interop_ = false;
```

Comment:
What's the reason behind needing this boolean vs. previous version?
Not a bad change at all but I'd like to better understand the extent of the changes.

Reply:
It seems that this boolean is never set to `true` anywhere actually.